### PR TITLE
Compatibility with Heroku CI

### PR DESCRIPTION
**Pull request by chrblabla**
PR for #5 to establish compatibility with Heroku CI.

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -29,7 +29,11 @@ unzip -qq -d "$BUILD_DIR/.awscli" /tmp/awscli-bundle.zip |& indent
 # dir out of the way. The installer uses virtualenv, which means it must be
 # installed in the final location - we have to install into /app.
 rm -rf /tmp/awscli
-[ -e $INSTALL_DIR ] && mv $INSTALL_DIR /tmp/awscli
+
+# Since $BUILD_DIR can be /app, we can only move $INSTALL_DIR if it's not the same.
+if [ "$INSTALL_DIR" != "$BUILD_DIR/.awscli" ]; then
```
Comment (dmathieu) [inline bin/compile]: How about having a single condition?
```
if [ "$INSTALL_DIR" != "$BUILD_DIR/.awscli" && -e $INSTALL_DIR ]; then
```

**Review by dmathieu — COMMENTED** [review]
This looks nice, thank you!

File: bin/compile (hunk 1 above)
Comment (chrblabla) [reply bin/compile]: Sure thing! Just wanted the diff to be easily readable. Update coming!

**Review by dmathieu — APPROVED** [review]
Thank you! 🌷

**Pull request merged**